internal/controller/storagemgr/storagedriver/tests.log

```diff
@@ -0,0 +1,34 @@
+level=ERROR ts=2021-01-21T08:47:29.592063566Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-21T08:47:29.598763268Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
+level=ERROR ts=2021-01-21T09:11:14.19516319Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-21T09:11:14.202153315Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
+level=ERROR ts=2021-01-21T09:22:51.122903185Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-21T09:22:51.128942501Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
+level=ERROR ts=2021-01-22T00:52:20.184210738Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-22T00:52:20.192300306Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
+level=ERROR ts=2021-01-22T00:54:08.752331582Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-22T00:54:08.757857483Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
+level=ERROR ts=2021-01-22T00:56:01.581135739Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-22T00:56:01.587816248Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
+level=ERROR ts=2021-01-22T00:58:04.286154926Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-22T00:58:04.291884501Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
+level=ERROR ts=2021-01-22T02:58:06.063788028Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-22T02:58:06.065130197Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
+level=ERROR ts=2021-01-25T02:50:32.205478144Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading -123.junk is out of the value type(19)'s range"
+level=ERROR ts=2021-01-25T02:50:32.206797153Z app=test source=storagehandler.go:207 msg="parse reading fail. Reading Random is out of the value type(21)'s range"
```

Comment:
Please remove the `internal/controller/storagemgr/storagedriver/tests.log` file from the tracking

Reply:
@tdrozdovsky Thank you very much for reviewing carefully and not merging unnecessary file!